src/content/en/fundamentals/performance/http2/index.md

```diff
@@ -254,7 +254,7 @@ reassemble them on the other end.
 ![HTTP/2 request and response multiplexing within a shared connection](images/multiplexing01.svg)
 
 The snapshot captures multiple streams in flight within the same connection. The
-client is transmitting a DATA frame (stream 5) to the server, while the server
+client is transmitting a `DATA` frame (stream 5) to the server, while the server
```

Comment:
IMHO the [RFC 7540 sec. 6](https://tools.ietf.org/html/rfc7540#section-6) specify `DATA` and `PUSH_PROMISE` frames meaning like markup code in HTML so I think the formatting is more correct with backticks. If I'm wrong I'll remove that from my commit.